mltmess/issue_fugue.py

```diff
@@ -0,0 +1,24 @@
+import pandas as pd
```

Comment:
What's mltmess

Reply:
Yek that's my mess!

Reply:
Removed in https://github.com/holoviz/hvplot/pull/1102/commits/7167264f7cc5469fe507703ea439ffa487f7c8a2